### PR TITLE
Prod local

### PR DESCRIPTION
**Pull request by code-blooded-guy**

### CODE REVIEW

File: sunbird-rc-ED-ui/src/app/forms/forms.component.ts [hunk 1]
```diff
@@ -1962,8 +1962,9 @@ export class FormsComponent implements OnInit {
               this.identifier,
             ];
           } else {
-            if (this.form == 'ag-registration' ||  this.form == 'ag-setup') {
 
+            if (this.form == 'ag-registration' ||  this.form == 'ag-setup') {
+              console.log("formmmm",this.form)
```
Comment (snehal0904) [inline sunbird-rc-ED-ui/src/app/forms/forms.component.ts]: @code-blooded-guy Remove console

File: sunbird-rc-ED-ui/src/app/forms/forms.component.ts [hunk 2]
```diff
@@ -1994,11 +2005,14 @@ export class FormsComponent implements OnInit {
             this.adminForm == 'prerak-admin-setup' ||
             this.adminForm == 'interview' ||
             this.form == 'ag-setup' ||
-            this.form == 'ag-registration' && this.add
+            this.form == 'ag-registration'
           ) {
-            this.postData();
-          } else {
-            this.updateClaims();
+            console.log("formmmm2",this.form)
```
Comment (snehal0904) [inline sunbird-rc-ED-ui/src/app/forms/forms.component.ts]: @code-blooded-guy Remove

File: sunbird-rc-ED-ui/proxy.conf.json [hunk 3]
```diff
@@ -1,21 +1,21 @@
 {
     "/registry/api/v1": {
-      "target": "https://dev-eg2.uniteframework.io",
+      "target": "https://eg2.uniteframework.io",
```
Comment (snehal0904) [inline sunbird-rc-ED-ui/proxy.conf.json:3]: @code-blooded-guy Remove this file from merge

Comment (code-blooded-guy) [reply sunbird-rc-ED-ui/proxy.conf.json:3]: This file will not reflect anywhere in deployment, it's only for local development

**Pull request merged**